src/styles/App.css

```diff
@@ -1,3 +1,16 @@
 .App {
   @apply relative;
 }
+/*****styling vertical scroll bar*****/
```

Comment:
I feel we don't need to change this as the catchy color will grab the user's attention a lot more to the scrollbar than required and would not be suitable for good the user experience,

Reply:
So like can i try to change the color to little lighter or something else , or no change is needed for this ?